src/styles/blocks/card.scss

```diff
@@ -0,0 +1,145 @@
+.card {
+
+  box-sizing: border-box;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+  background: #FFF;
+
+  padding: 32px 16px 16px;
+
+  &:hover {
+    transition: all .3s;
+    transform: scale(1.2);
+  }
+  &:not(:hover) {
+    transition: all .3s;
+    transform: scale(1);
```

Comment:
```suggestion
```
This part is redundant, no need to hardcode it. Correct transition will do it automatically